RealmWeaver/ModuleWeaver.cs

```diff
@@ -178,11 +178,12 @@ public void Execute()
         var listTypeDefinition = _corLib.MainModule.GetType("System.Collections.Generic.List`1");
         if (listTypeDefinition == null)
         {
-            var collectionsAssembly = AssemblyResolver.Resolve("System.Collections");
-            listTypeDefinition = collectionsAssembly.MainModule.ExportedTypes.FirstOrDefault(x => x.FullName == "System.Collections.Generic.List`1").Resolve();
-            
+            System_IList = _realmAssembly.MainModule.ImportReference(typeof (System.Collections.Generic.List<>));
```

Comment:
Shouldn't this be importing the type reference in the `ModuleDefinition` of the assembly we're currently weaving, as opposed to the Realm assembly?